### PR TITLE
Rxn dfn

### PR DESCRIPTION
**Pull request by riveSunder**
adds new gliders, scalable neighborhood kernel (with Laplacian of Gaussian kernel), and associated changes

However, gliders are not stable at kernel sizes other than 3x3!

### CODE REVIEW

File: coverage.txt [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
 yuca/wrappers/__init__.py              3      0   100%
 yuca/wrappers/glider_wrapper.py       90     13    86%
 yuca/wrappers/halting_wrapper.py     184     29    84%
 yuca/wrappers/random_wrapper.py       18      5    72%
+yuca/zoo/__init__.py                   0      0   100%
 ------------------------------------------------------
-TOTAL                               2079    295    86%
-message: revert glider_wrapper changes per review
\ No newline at end of file
+TOTAL                               2127    313    85%
```
Comment (riveSunder) [inline coverage.txt]: ✓

File: yuca/activations.py [hunk 2]
```diff
@@ -151,7 +151,27 @@ def forward(self, x):
...
+
+        if type(x) is not torch.Tensor:
+            x = torch.tensor(x)
+
+        laplacian_of_gaussian = -(1 / (torch.pi * self.sigma**4)) \
+                * (1 - (x**2) / (2*self.sigma**2)) \
+                * (torch.exp(- (x**2)/(2*self.sigma**2)))
+
+        laplacian_of_gaussian -= laplacian_of_gaussian.mean()
+
```
Comment (riveSunder) [inline yuca/activations.py:173]: normalize by sum of absolute value?

File: coverage.txt [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
 yuca/wrappers/glider_wrapper.py       90     13    86%
 yuca/wrappers/halting_wrapper.py     184     29    84%
 yuca/wrappers/random_wrapper.py       18      5    72%
+yuca/zoo/__init__.py                   0      0   100%
 ------------------------------------------------------
-TOTAL                               2079    295    86%
-message: revert glider_wrapper changes per review
\ No newline at end of file
+TOTAL                               2128    313    85%
+message: scale laplacian of gaussian kernels by sum of absolute value
```
Comment (riveSunder) [inline coverage.txt:24]: ✓

**Review by riveSunder — COMMENTED** [review]
ok to merge
